js/renovation/ui/toolbar/toolbar.tsx

```diff
@@ -8,18 +8,31 @@ import LegacyToolbar from '../../../ui/toolbar';
 
 import { DomComponentWrapper } from '../common/dom_component_wrapper';
 import { ToolbarProps } from './toolbar_props';
+import { isDefined } from '../../../core/utils/type';
 
 export const viewFunction = ({
   props,
   restAttributes,
-}: Toolbar): JSX.Element => (
-  <DomComponentWrapper
-    componentType={LegacyToolbar}
-    componentProps={props}
-  // eslint-disable-next-line react/jsx-props-no-spreading
-    {...restAttributes}
-  />
-);
+}: Toolbar): JSX.Element => {
+  const { rtlEnabled, items } = props;
+
+  items?.forEach((item) => {
+    if (typeof item !== 'string') {
```

Comment:
We have helper method isString?

Reply:
Why do you ignore string defined item? It's also should implement the same rtlEnabled behaviour. Isn't it?

Reply:
No, if item is string, then it will be rendered as pure string inside the Toolbar.
Because Toolbar has class rtl, string will be rendered from right to left

Reply:
isString is fixed